aten/src/TH/THTensor.hpp

```diff
@@ -9,6 +9,11 @@
 #include <atomic>
 #include <ATen/ATen.h>
 
+inline at::Tensor THTensor_wrap(THTensor* tensor) {
```

Comment:
Needs a doc; specifically, the doc should say that `THTensor* tensor` stays valid even after this call.